### PR TITLE
Add Note Display & User guide add-on

### PR DESCRIPTION
**Pull request by Nazryl**
#180
ISPINNED will display boolean instead of string

#172
UG add-ons

### CODE REVIEW

File: src/main/java/seedu/notus/ui/Formatter.java [hunk 1]
```diff
@@ -115,7 +115,7 @@ public static String formatNote(String message, Note note) {
         String header = message.concat(note.getTitle() + " " + note.getTagsName());
 
         if (note.getPinned()) {
-            header = header.concat("| ISPIN: " + note.getPinnedString() + " ");
+            header = header.concat("| ISPIN: " + note.getPinned() + " ");
```
Comment (Chongjx) [inline src/main/java/seedu/notus/ui/Formatter.java:118]: Suggestion for the message to just displayed "pinned" or "unpinned" without the " | ISPIN: ". Likewise for archive, can display "Archived" for archive notes

Comment (Nazryl) [reply src/main/java/seedu/notus/ui/Formatter.java:118]: Ramana suggested both to be true false.

I think ill display pinned" or "unpinned and archived when its archive. But when its unarchive i wouldn't show it.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -101,6 +101,7 @@ Adds a new note to the list of note items (think of it as a notebook).
 Format: `add-n /t TITLE [/tag TAG_1] [/tag TAG_2]... [/pin ISPIN] [/archive ISARCHIVE]`
 
 💡 Each note has to have a **UNIQUE** title and it is **CASE-INSENSITIVE**. <br>
+💡 New notes cannot have the same title as those in the archives. <br>
```
Comment (R-Ramana) [inline docs/UserGuide.md:104]:
```suggestion
💡 Each note has to have a **UNIQUE** title and it is **CASE-INSENSITIVE**. <br>
💡 As an extension to the above point, new notes cannot have the same title as those archived. <br>
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -200,6 +201,7 @@ Format: `edit-n /i INDEX [/t TITLE] ([/add INDEX STRING] OR [/ln LINE_INDEX CONT
 - Existing values will be updated to the input values.
 - When editing tags, the existing tags will be removed, while the non-existing tags will be added.
 
+💡 Note cannot have the same title as those in the archives. <br>
```
Comment (R-Ramana) [inline docs/UserGuide.md:204]:
```suggestion
💡 Each note has to have a **UNIQUE** title and it is **CASE-INSENSITIVE**. <br>
💡 As an extension to the above point, new notes cannot have the same title as those archived. <br>
```

**Review by R-Ramana — APPROVED** [review]

**Review by R-Ramana — APPROVED** [review]

File: src/main/java/seedu/notus/data/notebook/Note.java [hunk 4]
```diff
@@ -121,6 +121,10 @@ public boolean getIsArchived() {
         return this.isArchived;
     }
 
+    public String getIsArchivedString() {
+        return (isArchived ? "Archived" : "Unarchived");
```
Comment (Chongjx) [inline src/main/java/seedu/notus/data/notebook/Note.java:125]: I feel that can return "" for unarchived, don't really need to state that a normal note is unarchived

**Pull request merged**